kafka_consumer/tests/test_kafka_consumer.py

```diff
@@ -178,13 +178,13 @@ def assert_check_kafka(aggregator, consumer_groups, e2e=False):
                     aggregator.assert_metric(mname, tags=tags, at_least=1)
                 for mname in CONSUMER_METRICS:
                     aggregator.assert_metric(mname, tags=tags + ["consumer_group:{}".format(name)], at_least=1)
-                if not is_legacy_check() and not e2e:
```

Comment:
I moved this conditional outside of the nested `for` loop since it's not dependent on anything in there.